src/components/pagination-nav/pagination-nav.js

```diff
@@ -46,20 +49,63 @@ const props = {
     type: Function,
     default: null
   },
+  noPageDetect: {
+    // Disable auto page number detection if true
+    type: Boolean,
+    default: false
+  },
   // Router specific props
   ...routerProps
 }
 
 // Our render function is brought in via the pagination mixin
 // @vue/component
 export default {
-  name: 'BPaginatonNav',
+  name: 'BPaginationNav',
```

Comment:
@tmorehouse Is this a breaking change?

Reply:
Nope... name is just there so that VueDevUtils can provide a component name when a component is instantiated in another component by reference to it's definition (rather than by kebab-case name)